base/uk.ac.stfc.isis.ibex.configserver/src/uk/ac/stfc/isis/ibex/configserver/configuration/Block.java

```diff
@@ -48,7 +48,7 @@
     private boolean runcontrol;
     private double lowlimit;
     private double highlimit;
-    private boolean suspend_on_invalid;
+    private boolean suspendOnInvalid;
```

Comment:
I'm not sure this will work, see note at top of class, the variable names must reflect those expected from the JSON.